### DISCUSSION TITLE
session.results of the Saudi Arabia

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#551] [Q&A] [answered]
Does it only happen to me?
Or the df session.results of the Saudi Arabia race is still not updated and with the data from the Q?

Thanks in advance.

**Comment by theOehrly** [comment, marked as answer]
Have you updated to the latest version of FastF1 (v3.3.1)? There was a bug that prevented the results from being loaded correctly.

**Comment by McHouse** [comment]
Ahhhhh... the same thing always happens to me... :facepalm:
Thanks theOehrly.

**Reply by theOehrly** [reply]
No worries, ideally there wouldn't be an updated needed to support the latest sessions in the first place 😅
